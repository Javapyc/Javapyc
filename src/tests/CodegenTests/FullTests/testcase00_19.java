class Main {
	public static void main(String[] argv) {
		int a = 8;
		int b = 3;
		System.out.println(a/b);
		System.out.println(-1+(a-b*5)/3);
		while (a >= b) {
			a = a * 2;
			b = b * 3;
			System.out.println(a);
			System.out.println(b);
		}
	}
}
