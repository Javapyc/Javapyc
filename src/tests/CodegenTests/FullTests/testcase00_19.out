2
-3
16
9
32
27
64
81
